Userland/Applications/FileManager/main.cpp

```diff
@@ -779,13 +781,23 @@ int run_in_windowed_mode(RefPtr<Core::ConfigFile> config, String initial_locatio
         },
         &tree_view);
 
+    auto tree_view_trash_action = GUI::Action::create(
+        "Move to &Trash", Gfx::Bitmap::load_from_file("/res/icons/16x16/delete.png"), [&](auto&) {
```

Comment:
Please at least put a FIXME here and note that we should add and use a trash can icon :)